packages/eslint-plugin/src/rules/no-tslint-comments.js

```diff
@@ -7,7 +7,7 @@ module.exports = createRule({
     type: 'problem',
     docs: {
       description: 'Forbid tslint:disable/tslint:enable comments after ESLint migration.',
-      recommended: 'error',
+      recommended: 'recommended',
```

Comment:
has to be changed after [this](https://github.com/typescript-eslint/typescript-eslint/pull/5251)